pkg/mkimage-raw-efi/make-raw

```diff
@@ -381,19 +381,25 @@ INITRD_IMG=/parts/initrd.img
 INSTALLER_IMG=/parts/installer.img
 # GRUB cfg override for our installer
 INSTALLER_GRUB_CFG=/parts/grub.cfg
-# IMX8 flash blob that needs write to image with 33k offset
+# IMX8 flash blob that needs to be written to image
 IMX8_BLOB=/parts/imx8-flash.bin
+# IMX8 configuration file with flashing parameters
+IMX8_CONF=/parts/imx8-flash.conf
 
 # offset in the GPT partition table from which we can start our numbering
 PART_OFFSET=0
 # starting sector for our portion of the disk
 CUR_SEC="$FIRST_PART_SEC"
 
-if [ -f "$IMX8_BLOB" ]; then
-  # For imx8 boards we need write boot blob with 33kb offset.
+if [ -f "$IMX8_BLOB" ] && [ -f "$IMX8_CONF" ]; then
+  # For imx8 boards we to need write boot blob with the specified offset
```

Comment:
It might be a typo or I don't understand meaning of this sentence. And is there any guidance for the specified offset or is it at you own risk? I'm asking because in my experience this offset is very important and wrong number can make device unbootable

Reply:
That's a typo. Thanks for the catch, I will fix it. Regarding the offset, it's specific for each board, the config file with this number shall be populated by bsp-imx package, which will take care to put the right value, so no user actions are needed.

Reply:
What if user wants to create his own config?

Reply:
Since we generate the full SD Card image from our build system, we need to build everything (u-boot + EVE artifacts) for each particular ARM platform we support. Currently, users cannot simply generate custom images, specially because u-boot it's tied to the board that should run on it due to ddr timing initialization, SoC specific boot code, device tree, etc. In case of i.MX8M platform, there is a difference in the bootloader offset (for SD Card images) for i.MX8MQ and i.MX8MP, that's why this conf file was used. However, it's not intended to user changes. If user wants to add a new platform, it must do it through pkg/bsp-imx.

Reply:
Okay, got it, thanks a lot!